sp_api/api/shipping/shippingV2.py

```diff
@@ -0,0 +1,627 @@
+import enum
+import urllib.parse
+from datetime import datetime
+
+from sp_api.base import Client, sp_endpoint, fill_query_params, ApiResponse
+
+
+class AmznShippingBusiness(str, enum.Enum):
+    AmazonShipping_US = "AmazonShipping_US"
+    AmazonShipping_IN = "AmazonShipping_IN"
+    AmazonShipping_UK = "AmazonShipping_UK"
+    AmazonShipping_UAE = "AmazonShipping_UAE"
+    AmazonShipping_SA = "AmazonShipping_SA"
+    AmazonShipping_EG = "AmazonShipping_EG"
+    AmazonShipping_IT = "AmazonShipping_IT"
+    AmazonShipping_ES = "AmazonShipping_ES"
+    AmazonShipping_FR = "AmazonShipping_FR"
+    AmazonShipping_JP = "AmazonShipping_JP"
+
+
+class Shipping(Client):
+    """
+    Shipping V2 SP-API Client
+    :link: 
+
+    Provides programmatic access to Amazon Shipping APIs.
+    """
+
+    amzn_shipping_business: AmznShippingBusiness = AmznShippingBusiness.AmazonShipping_UK
+
+    def __init__(self, *args, **kwargs):
+        if 'amzn_shipping_business' in kwargs:
+            self.amzn_shipping_business = kwargs.pop('amzn_shipping_business', AmznShippingBusiness.AmazonShipping_UK)
```

Comment:
The library's default is US, this should be consistent. 

Cant the marketplace be taken from the client, like with all other endpoints?

Reply:
The default in Amazon shipping business is UK according to the documentation but I don't think it's bad to change it for US for consistency. 

About the marketplace, not all countries have Amazon Shipping business ID. What do you propose?

Reply:
In the documentation, you can see that the default is UK, and that there is no correspondence with all marketplaces. In fact you do not need to be an Amazon Marketplaces Seller in EU ([doc](https://developer-docs.amazon.com/amazon-shipping/docs/off-amazon-guide))

[x-amzn-shipping-business-id](https://developer-docs.amazon.com/amazon-shipping/docs/shipping-api-v2-reference#x-amzn-shipping-business-id)

What changes do you propose?

- I can change the default from UK to US.
- I can deduce your x-amzn-shipping-business-id from your marketplaces if you wish... And in case of non-matching, I can put the default.

Reply:
Sounds good. Let's do it as you see fit